modules/opencensus/pom.xml

```diff
@@ -0,0 +1,97 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.spotify.ffwd</groupId>
+    <artifactId>ffwd-parent</artifactId>
+    <version>0.5.6-SNAPSHOT</version>
+    <relativePath>../../pom.xml</relativePath>
+  </parent>
+
+  <artifactId>ffwd-module-opencensus</artifactId>
+  <version>0.5.6-SNAPSHOT</version>
```

Comment:
ah looks like the build is failing because this got changed to `<version>0.5.7-SNAPSHOT</version>`. i'll merge once that is fixed.

Reply:
Hiya, sorry about that, my git is a little rusty so I forgot to pull before I pushed.